docs/mp/guides/40_performance-tuning.adoc

```diff
@@ -0,0 +1,89 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Performance Tuning
+:h1Prefix: MP
+:description: Helidon MP Performance Tuning
+:feature-name: Performance Tuning
+:microprofile-bundle: true
+= :keywords: helidon, microprofile, micro-profile, performance, tuning
+
+
+== Use `io.helidon.microprofile.bundles:helidon-microprofile-core`
+
+Use `helidon-microprofile-core` dependency (and not the `helidon-microprofile` dependency) and add only what you use. For example:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.bundles</groupId>
+    <artifactId>helidon-microprofile-core</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.microprofile.metrics</groupId>
+    <artifactId>helidon-microprofile-metrics</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.microprofile.health</groupId>
+    <artifactId>helidon-microprofile-health</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.media</groupId>
+    <artifactId>helidon-media-jsonp</artifactId>
+</dependency>
+----
+
+In this guide you fill find basic advises for performance tuning of your Helidon application. Most of them target Netty tuning, as Helidon is based on it.
```

Comment:
advises -> advice

Also this paragraph needs to be moved before the previous section (like it is in the SE guide).

Reply:
Resolved)